cmd/agentflow/example-config.flow

```diff
@@ -1 +1,5 @@
+log_level  = "debug"
+log_format = "logfmt"
```

Comment:
Should we have a logging block instead of separate fields?

```hcl 
logging {
  level  = "debug"
  format = "logfmt"
}
```

Reply:
The block feels better to me. Also eventually the agent should expose its own logger and logging have an output makes logical sense.

Reply:
Any opinions on the block name? `logging`/`logger`/something else?